stable/graylog/templates/statefulset.yaml

```diff
@@ -149,6 +131,9 @@ spec:
             httpGet:
               path: /api/system/lbstatus
               port: 9000
+            {{- if .Values.graylog.ingress.tls }}
+              scheme: HTTPS
```

Comment:
This is internal ping by k8s, https won't be available.

Reply:
It will use the cluster ip of the pod for the request and when graylog is running in https mode, there is not http handler to handle the request so it fails and is never marked ready.

I am having a similar issue with the publish uri that I’m trying to solve right now. Because we don’t know the ips ahead of time, I cannot include that in the cert so the node to node communication is broken. I think I’m going to solve it by dynamically setting the publish uri in the startup script to the host name and generate the cert with enough sans to cover myself, but still a little janky. I don’t think that setting the publish uri equal to the external uri works when it’s a fqdn

Reply:
This is no correct. When using `ingress.tls` the TLS will be terminated at ingress layer. Graylog does not handle TLS at all. The k8s control plane will `httpGet` directly to Pods address and it will be failed since Graylog does only `http`.

Reply:
Ah yes, maybe we need to have a separate value for this use case. I'm talking about the case when you are not using an ingress and are running [Graylog in HTTPS mode](https://github.com/helm/charts/tree/master/stable/graylog#input-tls). I'll revert these changes here and open up a new PR with those changes and incorporate in that configuration so that there is less for the user to do.